rpc/client.ts

```diff
@@ -37,7 +37,18 @@ export class Client<
   }
 
   #listener: ProviderListener<SendErrorData, HandlerErrorData> = (e) => {
-    if (e instanceof Error) {
+    if (e instanceof ProviderSendError && e.egressMessage) {
+      const egressMessageId = e.egressMessage.id;
+      const pendingCall = this.pendingCalls[egressMessageId];
+      if (!pendingCall) {
+        console.log({ e });
+        // TODO: pipe error to listeners and message the likely cause,
+        //       a duplicate client.
+        throw new Error();
+      }
```

Comment:
I think this check can be removed.
It happens when the provider is being discarded and an inflight/pending response is emitted.
In that scenario, I assume that we don't care about the response so we can ignore it.

Thoughts @harrysolovay?

Reply:
Which lines specifically?

Reply:
```
if (!pendingCall) {
        console.log({ e });
        // TODO: pipe error to listeners and message the likely cause,
        //       a duplicate client.
        throw new Error();
      }
```

Reply:
Gotcha. Removing sounds good.